README.md

```diff
@@ -598,6 +601,11 @@ resolve_response = fullcontact_client.identity.resolve(email="marquitaross006@gm
 print(resolve_response.get_recordIds())
 # Output: ['customer123']
 
+# Synchronous mapResolve execution
+mapResolve_response = fullcontact_client.identity.mapResolve(email="marquitaross006@gmail.com", recordId="customer123")
+print(mapResolve_response.get_recordIds())
+# Output: ['customer123']
```

Comment:
Do we want to call out here that you can also get personIds? with get_personIds()? Looks like we don't have much example code with this being called.